### PR TITLE
proposed change to 4572

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-jennings-4572bis.md [hunk 1]
```diff
@@ -292,32 +313,61 @@ UHEX                   =  DIGIT / %x41-46 ; A-F uppercase
...
-consideration) MUST use one of the FIPS 180 algorithms (SHA-1,
-SHA-224, SHA-256, SHA-384, or SHA-512) as their signature algorithm,
-and thus also MUST use it to calculate certificate fingerprints.
 
 The fingerprint attribute may be either a session-level or a media-
 level SDP attribute.  If it is a session-level attribute, it applies
 to all TLS sessions for which no media-level fingerprint attribute is
 defined.
 
+ Multiple SDP fingerprint attributes can be associated with an m-
```
Comment (martinthomson) [inline draft-jennings-4572bis.md]: I think that we should add a section header here "## Multiple Fingerprints"

File: draft-jennings-4572bis.md [hunk 2]
```diff
@@ -494,8 +544,18 @@ TLS is not always the most appropriate choice for secure connection-
...
 
+This document improves security from the RFC 4572 [@RFC4572].  It
+updates the preferred hash function cipher suite from SHA-1 to
+SHA-256.  By clarifying the usage and handling of multiple
+fingerprints, the document also enables hash agility, and incremental
+deployment of newer, and more secure, cipher suites.
+
 #  IANA Considerations
 
+Note to IANA. No IANA considerations are changes from RFC4572
```
Comment (martinthomson) [inline draft-jennings-4572bis.md]: s/changes/changed/

File: draft-jennings-4572bis.md [hunk 3]
```diff
@@ -292,32 +313,61 @@ UHEX                   =  DIGIT / %x41-46 ; A-F uppercase
...
+ Multiple SDP fingerprint attributes can be associated with an m-
+ line. This can occur if multiple fingerprints have been calculated
+ for a certificate using different hash functions. It can also occur
+ if one or more fingerprints associated with multiple certificates
+ have been calculated. This might be needed if multiple certificates
+ will be used for media associated with an m- line (e.g. if separate
+ certificates are used for RTP and RTCP), or where it is not known
+ which certificate will be used when the fingerprints are
+ exchanged. In such cases, one or more fingerprints MUST be calculated
+ for each possible certificate. An endpoint MUST, as a minimum,
```
Comment (martinthomson) [inline draft-jennings-4572bis.md]: I would break this mega-paragraph before "An endpoint MUST..."

File: draft-jennings-4572bis.md [hunk 4]
```diff
@@ -3,8 +3,9 @@
     abbrev = "Comedia over TLS in SDP"
     category = "std"
     docName = "draft-jennings-4572bis-00"
-    ipr= "trust200902"
+    ipr = "trust200902"
     area = "ART"
+    obsoletes = [ 4572 ]
 
     [pi]
     symrefs = "no"
```
Comment (martinthomson) [inline draft-jennings-4572bis.md]: FWIW, I would prefer symrefs = "yes"

Comment (fluffy) [inline draft-jennings-4572bis.md]: I agree and happy to do that at some point but for now I left it as close to RFC4572 as possible to minimize changes.

**Pull request merged**